homework3/task1/task1.cpp

```diff
@@ -0,0 +1,36 @@
+﻿#include <iostream>
+
+using namespace std;
+
+double powerNumber(double number, double power);
```

Comment:
Your function returns some kind of result so give one name starting with a verb

Reply:
![image](https://user-images.githubusercontent.com/59542254/111069179-f8656c00-84dc-11eb-9930-9092f4d073b4.png)

Reply:
I think getPowerNumber more suitable

Reply:
done